docs/components/select-rich/use-cases.md

```diff
@@ -188,15 +188,15 @@ export const checkedIndexAndValue = ({ shadowRoot }) => html`
       aria-labelledby="label-richSelectCheckedInput"
       type="number"
       @change=${e => {
-        const selectEl = shadowRoot.getElementById('checkedRichSelect');
+        const selectEl = shadowRoot.querySelector('#checkedRichSelect');
         selectEl.checkedIndex = e.target.value;
       }}
     />
   </div>
   <button
     class="log-button"
     @click=${() => {
-      const selectEl = shadowRoot.getElementById('checkedRichSelect');
+      const selectEl = shadowRoot.querySelector('#checkedRichSelect');
       console.log(`checkedIndex: ${selectEl.checkedIndex}`);
       console.log(`checkedValue: ${selectEl.checkedValue}`);
```

Comment:
CheckedValue is something from the past, should be `modelValue`.

Reply:
fixed.